data/remnant/remnant 2 side missions.txt

```diff
@@ -715,3 +715,59 @@ mission "Remnant: Scent of Rain 1"
 			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
 			label scanningpayment
 			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
+
+
+
+mission "Remnant: Will Not Someone Please Think Of The Children"
+	landing
+	name "Sympathy For The Korath"
+	description "Travel to the <waypoints> system to disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
+	minor
+	source
+		government "Remnant"
+		not attributes "station"
```

Comment:
I don't think an object exists with the Remnant government and the `station` attribute.
Ssil Vida has the `remnant station` attribute, but it seems like the desire is for this mission to be available on Ssil Vida after a certain point.

Reply:
Chilia on Ssil Vida seemed weird to me. 

Also I was actually kind of worried about all the sector modification logic that happens with Ssil Vida. So a little bit of fear of the unknown also and reduces possible future issues.

Reply:
The current `to offer` conditions say to me this mission should have the possibility of being offered on Ssil Vida from the end of the Remnant chapter 2 story onwards. If this mission should not be offered there, then the `to offer` conditions should be simplified and this location filter should be changed to `not attributes "remnant station"` since that's the attribute Ssil Vida has.
The attributes of Ssil Vida are not changes at any point in the existing events.
Regardless, `not attributes "station"` is just confusing since its presence doesn't do anything unless I've missed a planet.

Reply:
Changed to allow to be offered on Ssil Vida, only source requirement now is Remnant government.